gazebo/rendering/Light.cc

```diff
@@ -156,14 +156,20 @@ void Light::UpdateSDFFromMsg(const msgs::Light &_msg)
 //////////////////////////////////////////////////
 void Light::UpdateFromMsg(ConstLightPtr &_msg)
 {
-  this->UpdateSDFFromMsg(*_msg);
+  this->UpdateFromMsg(*_msg);
```

Comment:
I think this `_msg` pointer should be checked before dereferencing it

Reply:
Resolved (it ignores null messages).